src/builtins/impls/units.rs

```diff
@@ -35,6 +46,6 @@ pub fn parse(x: String) -> Result<i64> {
 /// units. The bytes symbol (b/B) in the unit is optional and omitting it wil
 /// give the same result (e.g. Mi and MiB).
 #[tracing::instrument(name = "units.parse_bytes", err)]
-pub fn parse_bytes(x: String) -> Result<i64> {
-    bail!("not implemented");
+pub fn parse_bytes(x: String) -> Result<u64> {
+    Ok(Config::new().with_decimal().parse_size(x.as_str())?)
```

Comment:
You don't have to change it here, but FYI there is a way to change a `Result<T, E>` to a `Result<T, anyhow::Error>` with a custom message with the `anyhow::Context` trait, to avoid the `Ok(foo?)` pattern:

```suggestion
    Config::new().with_decimal().parse_size(x.as_str()).context("could not parse value")
```